gosrc/gosrc.go

```diff
@@ -186,7 +186,7 @@ func attrValue(attrs []xml.Attr, name string) string {
 	return ""
 }
 
-func fetchMeta(client *http.Client, importPath string) (string, *importMeta, *sourceMeta, error) {
+func fetchMeta(client *http.Client, importPath string) (string, *importMeta, *sourceMeta, bool, error) {
```

Comment:
This function needs named return parameters; they other types were self-explanatory but 'bool' is not.